api/http/management.go

```diff
@@ -105,7 +105,7 @@ func (h *ManagementHandler) CreateIntegration(c *gin.Context) {
 		}
 		return
 	}
-	c.Status(http.StatusNoContent)
+	c.JSON(http.StatusOK, inserted)
```

Comment:
it's not strictly related to this PR, since this PR contains implementation, not the API spec, but I have one comment - I think this is a new pattern in our APIs; usually, we're returning 201 Created with Location header;
it's ok to return 200 and the created object, but it introduces some inconsistency in our API, and if we want to do it this way, we should probably also return "Content Location" header

Reply:
This is how @alfrunes designed the API, I implemented it according to the API docs.
Anyway, I agree with you, let me rework it.

Reply:
@kjaskiewiczz done, see the updated commits